int32 model_id
string name
float32 score
geometry_msgs/PoseStamped pose
geometric_shapes_msgs/Shape mesh
